app/src/main/java/ch/epfl/balelecbud/view/emergency/EmergencyInformationData.java

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.balelecbud.view.emergency;
+
+import java.util.LinkedList;
+
+import ch.epfl.balelecbud.BalelecbudApplication;
+import ch.epfl.balelecbud.model.EmergencyInformation;
+import ch.epfl.balelecbud.utility.CompletableFutureUtils;
+import ch.epfl.balelecbud.utility.database.Database;
+import ch.epfl.balelecbud.utility.database.query.MyQuery;
+import ch.epfl.balelecbud.utility.recyclerViews.OnRecyclerViewInteractionListener;
+import ch.epfl.balelecbud.utility.recyclerViews.RecyclerViewData;
+
+public class EmergencyInformationData extends RecyclerViewData<EmergencyInformation, EmergencyInformationHolder> {
```

Comment:
This class is very similar to EmergencyNumbersData, is it possible to delete it? You could setOnClickListener only when needed using data.get(index).isEmergencyNumber()